packages/app/src/cli/services/app_logs/fetch-app-logs.ts

```diff
@@ -0,0 +1,95 @@
+import {writeAppLogsToFile} from './write-app-logs.js'
+import {partnersFqdn} from '@shopify/cli-kit/node/context/fqdn'
+import {fetch} from '@shopify/cli-kit/node/http'
+import {Writable} from 'stream'
+
+const pollingInterval = 450
```

Comment:
nit, would be good to indicate this is ms, either in a comment or the variable name

Reply:
👍 I can add `Ms` at the end. In `setTimeout` it's just `ms?: number,`